services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: None = None) -> None:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: None) -> None:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: None,
+        show_time: None = None,
+        movie_id: None = None,
+        cinema_hall_id: None = None

```

Comment:
The parameters `session_id`, `show_time`, `movie_id`, and `cinema_hall_id` should have appropriate types instead of `None`. Consider using `int` for `session_id`, `datetime` for `show_time`, and `int` for `movie_id` and `cinema_hall_id`.